src/components/CardChartContent.js

```diff
@@ -5,10 +5,11 @@ import Box from '@material-ui/core/Box';
 import BarChart from './BarChart';
 import PieChart from './PieChart';
 
+import PropTypes from 'prop-types';
```

Comment:
Remember that the absolute imports should always precede the relative ones. This should be near the top